docs-site/docs/user/project/samples/manage-samples.md

```diff
@@ -95,11 +100,23 @@ Prerequisites:
 
 To remove a sample:
 
-1. From the left sidebar, select **Projects**, select the project
-2. From the left sidebar, select **Samples**
-3. Select the sample that you would like to remove
-4. Select **Remove**
+1. From the left sidebar, select **Projects**
+2. Select the project
+3. From the left sidebar, select **Samples**
+4. Select the sample
+5. Select **Remove**
 
 ## View Sample History
 
-<!-- TODO: Add steps for sample history -->
+Prerequisite:
+- You must have at least the Maintainer role for the sample's project.
```

Comment:
Same comment as above about only direct or inherited members can see the history

Reply:
Added in e224fa502c5bc1af0101a56f5be2e42f97937f52